README

```diff
@@ -0,0 +1,85 @@
+Pg-Reindex
```

Comment:
I'd really prefer to have a `README.md` which is generated from the POD in the module. This boilerplate just tells how to install it but not what it is.

The README on github should first and foremost explain what this project is about.